src/index.js

```diff
@@ -122,14 +127,16 @@ export const TypographyProvider = ({
 
 
   return (
-    <ThemeProvider theme={systemTheme}>
-      <GoogleFont theme={theme} />
-      <style
-        dangerouslySetInnerHTML={{
-          __html: styles
-        }}
-      />
-      <Root {...props} />
-    </ThemeProvider>
+    <TypographyContext.Provider value={theme}>
+      <ThemeProvider theme={systemTheme}>
```

Comment:
Think one of these could go away if this uses the same `ThemeContext` from `@emotion/core` – also, I think the `systemTheme` should be the one provided as a value in context